### PR TITLE
chore(releasing): Prepare v0.41.0 release

### PR DESCRIPTION
**Pull request by jszwedko**
Checklist: https://github.com/vectordotdev/vector/issues/21228

### CODE REVIEW

**Review by jszwedko — COMMENTED** [review]
/ci-run-all

**Review by pront — APPROVED** [review]
Mostly looks good, there are a few typos.

File: website/cue/reference/releases/0.41.0.cue [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+			description: """
+				The `socket` source now respects the global `log_namespace` setting.
+				"""
+			contributors: ["Zettroke"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Windows now supports the `--watch-config` command line parameter just like every
+				other platform and will reload the configuration files on any change
```
Comment (alai97) [inline website/cue/reference/releases/0.41.0.cue:217]:
```suggestion
				other platform and will reload the configuration files on any change.
```

File: website/cue/reference/releases/0.41.0.cue [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+			type: "fix"
+			description: """
+				The `reduce` transform can now reduce fields that contain special characters.
+				"""
+		},
+		{
+			type: "fix"
+			description: """
+				Vector no longer panics during configuration loading if a secret is used for
+				a configuration option that has additional validation (e.g. URIs).
```
Comment (alai97) [inline website/cue/reference/releases/0.41.0.cue:187]:
```suggestion
				a configuration option that has additional validation (such as URIs).
```

File: website/cue/reference/releases/0.41.0.cue [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+				The `kafka` sink no longer emits warnings due to applying rdkafka options to
+				a consumer used for the health check. Now it uses the producer client for the health
+				check.
+				"""
+			contributors: ["belltoy"]
+		},
+		{
+			type: "fix"
+			description: """
+				The `amqp` source no longer panics when deserializing metrics (e.g. via the
```
Comment (alai97) [inline website/cue/reference/releases/0.41.0.cue:165]:
```suggestion
				The `amqp` source no longer panics when deserializing metrics (such as the
```

File: website/cue/reference/releases/0.41.0.cue [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+				Log sources can now use metrics-only decoders such as the recently added `influxdb` decoder.
+				"""
+			contributors: ["jorgehermo9"]
+		},
+		{
+			type: "fix"
+			description: """
+				The `influxdb` decoder now uses nanosecond-precision for timestamps instead of
+				microsecond-precision, as stated in [InfluxDB's
+				documentation](https://docs.influxdata.com/influxdb/v1/write_protocols/line_protocol_tutorial/#timestamp)
```
Comment (alai97) [inline website/cue/reference/releases/0.41.0.cue:142]:
```suggestion
				documentation](https://docs.influxdata.com/influxdb/v1/write_protocols/line_protocol_tutorial/#timestamp).
```

**Review by alai97 — APPROVED** [review]
Looks good, some consistency nits.

**Pull request merged**